app/src/main/java/com/smelnykov/homework_1/AuthActivity/AuthActivity.kt

```diff
@@ -13,13 +15,21 @@ import com.smelnykov.homework_1.databinding.ActivityAuthBinding
  */
 class AuthActivity : AppCompatActivity() {
 
-    private lateinit var binding : ActivityAuthBinding
+    private lateinit var binding: ActivityAuthBinding
+    private lateinit var sharedPreference: SharedPreferences
```

Comment:
lazy is generally safer to use than lateinit

Reply:
Thank you! Made changes. Commit 9e0bcee7